docs/getting-started/security-ui.asciidoc

```diff
@@ -137,12 +145,24 @@ image::images/network-ui.png[Shows the Network page]
 image::images/users/users-page.png[Shows the Users page]
 
 [float]
-=== Intelligence
+=== Rules
 
-The Intelligence section contains the Indicators page, which collects data from enabled threat intelligence feeds and provides a centralized view of indicators of compromise (IoCs). Refer to <<indicators-of-compromise>> to learn more.
+Expand this section to view the following pages:
 
+* <<rules-ui-create, *Rules*>>: Create and manage rules to monitor suspicious events.
```

Comment:
The *Rules* -> *Detection rules (SIEM)* nav link leads to the Rule Management page. Shouldn't we link to [`guide/en/security/master/rules-ui-management.html`](https://www.elastic.co/guide/en/security/master/rules-ui-management.html) here accordingly?

Reply:
That makes sense to me, thanks @banderror!
Resolved in [d6bf835](https://github.com/elastic/security-docs/pull/3785/commits/d6bf835092f07d2f7d7e2b6a115ba9d8ffbd069e).